src/Tasks/Microsoft.NET.Build.Tasks/Resources/Strings.resx

```diff
@@ -183,4 +183,46 @@
   <data name="InvalidNuGetVersionString" xml:space="preserve">
     <value>Invalid NuGet version string: '{0}'.</value>
   </data>
+  <data name="DOTNET1011" xml:space="preserve">
+    <value>Framework not installed: {0} in {1}</value>
+  </data>
+  <data name="DOTNET1012" xml:space="preserve">
+    <value>The reference assemblies directory was not specified. You can set the location using the DOTNET_REFERENCE_ASSEMBLIES_PATH environment variable.</value>
```

Comment:
It is $(TargetFrameworkRootDirectory) in msbuild, but we also get in other code paths from DOTNET_REFERENCE_ASSEMBLIES_PATH via a CLI API. We're busted for desktop builds on Mono right now with the mismatch. I just so happen to be looking at fixing this while I'm on vacation, but I think the expedient approach (only SDK changes) involves respecting both.